src/signature.rs

```diff
@@ -184,6 +184,12 @@ pub use rsa::{
     RSA_PKCS1_3072_8192_SHA384,
 };
 
+/// Lower-level verification primitives. Usage of the `signature::verify()`
+/// function is preferred.
```

Comment:
I suggest: "Lower-level verification primitives. Usage of `ring::signature::verify()` is preferred when the public key and signature is encoded in standard formats, as it also handles the parsing."